src/dask_awkward/layers/layers.py

```diff
@@ -117,6 +117,19 @@ def io_func_implements_report(func: ImplementsIOFunction) -> bool:
     return hasattr(func, "return_report")
 
 
+class AwkwardTokenizable:
+
+    def __init__(self, ret_func, parent_name):
+        self.parent_name = parent_name
+        self.ret_func = ret_func
+
+    def __dask_tokenize__(self):
+        return (AwkwardTokenizable, self.parent_name)
```

Comment:
May as well 
```suggestion
        return ("AwkwardTokenizable", self.parent_name)
```
or other literal we make just once